signer/tests/integration/postgres.rs

```diff
@@ -1362,7 +1364,197 @@ async fn get_signers_script_pubkeys_returns_non_empty_vec_old_rows() {
     signer::testing::storage::drop_db(db).await;
 }
 
+/// This tests that deposit requests where there is an associated sweep
+/// transaction will show up in the query results from
+/// [`DbRead::get_swept_deposit_requests`].
+#[cfg_attr(not(feature = "integration-tests"), ignore)]
+#[tokio::test]
+async fn get_swept_deposit_requests_returns_swept_deposit_requests() {
+    let db_num = DATABASE_NUM.fetch_add(1, Ordering::SeqCst);
+    let db = testing::storage::new_test_database(db_num, true).await;
+    let mut rng = rand::rngs::StdRng::seed_from_u64(51);
+
+    // This query doesn't *need* bitcoind (it's just a query), we just need
+    // the transaction data in the database. We use the [`TestSweepSetup`]
+    // structure because it has helper functions for generating and storing
+    // sweep transactions, and the [`TestSweepSetup`] structure correctly
+    // sets up the database.
+    let (rpc, faucet) = sbtc::testing::regtest::initialize_blockchain();
+    let setup = TestSweepSetup::new_setup(&rpc, &faucet, 1_000_000, &mut rng);
+
+    // Wwe need to manually update the database with new bitcoin block
```

Comment:
```suggestion
    // We need to manually update the database with new bitcoin block
```

Reply:
Done in https://github.com/stacks-network/sbtc/pull/615/commits/97dda8d25209b504926158549605fa0698ba0d3b